fix(clerk-js): Fix captcha layout shift on invisible and custom flows (#5049)
clerk/javascript: .changeset/fast-kiwis-visit.md

---
'@clerk/clerk-js': patch
---

> Fix layout shift on transfer flow when captcha is enabled

Author: ```suggestion
Fix captcha layout shift on transfer flow, custom flows and invisible
```
